### PR TITLE
feat: add image

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

**Review by VishalPatil18 — COMMENTED** [review]
Hey, Savita the code looks good but the image.html page is not opening in the netlify preview. You might check what is the issue. It shows `Page not Found`

**Comment by vermasavita** [comment]
@VishalPatil18 Maybe because that's not merged yet.

**Review by VishalPatil18 — APPROVED** [review]

File: image/image.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.img-box-1 img {
+    height: 40rem;
+    width: 100%;
+    overflow: hidden;
+    object-fit: cover;
+
```
Comment (ruchiragoswami) [inline image/image.css]: Just a small suggestion, there is an extra empty line.

**Review by ruchiragoswami — APPROVED** [review]
Looks good Savita! Great work!

File: image/image.css (hunk 1 above)
Comment (vermasavita) [reply image/image.css]: Removed.

**Pull request merged**